src/requireAccess.ts

```diff
@@ -47,9 +45,9 @@ module.exports = function(options) {
                 "To use " +
                 clc.bold(realProjectId) +
                 " instead, run:\n\n  " +
-                clc.bold(fixCommand)
-            ),
-            { exit: 1 }
+                clc.bold(fixCommand),
```

Comment:
Looks like this was a bug before - when I turned this into TS it complained the number of args given to `Promise.reject()` was one too many. Indeed, the parens needed to be moved to make `{exit:1}` be the option argument.